sass/_global.scss

```diff
@@ -22,3 +22,26 @@ body {
 .region-sidebar-first {
   padding: 1.667em 0;
 }
+
+.navbar-toggle {
+  border-radius: 0;
+  border: none;
+}
+
+.navbar-collapse,
+.navbar-fixed-top,
+.navbar {
+  border: none;
+}
+
+.facetapi-facetapi-checkbox-links {
```

Comment:
mmmh....how can something like facetapi-facetapi-checkbox-links be part of our "global" sass file?
This is not a "default" component, it comes out of an implementation that we are not providing in the europa theme and it uses drupal classes and not anything "compatible" with our naming convention.